pygfx/utils/color.py

```diff
@@ -255,6 +255,20 @@ def css(self):
         else:
             return f"rgba({int(255*r+0.5)},{int(255*g+0.5)},{int(255*b+0.5)},{a:0.3f})"
 
+    def multiply_scalar(self, factor):
+        """Multiply the color by a scalar. The alpha channel is ignored."""
+        self._val[0] *= factor
+        self._val[1] *= factor
+        self._val[2] *= factor
+        return self
```

Comment:
I updated the code.

But I want to explain a little. The reason why this function was added before is that in the constructor of Color, the rgba component is truncated to a maximum of 1.0.

However, in some cases, values greater than 1.0 are meaningful. In some lighting BSDF functions, the color of the light is divided by `PI` as irradiance. In order to get the artist friendly lighting (not physically correct, but more colorful and brighter), when passing the light color to the shader,  it will be multiplied by `PI` as compensation.

See: https://github.com/mrdoob/three.js/blob/085222f7ca0de96ab362a8b048235bf7187ebc08/src/renderers/webgl/WebGLLights.js#L220-L221